target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -165,6 +165,11 @@ wn3000rpv3)
 	ucidef_set_led_default "power_r" "POWER (red)" "$board:red:power" "0"
 	set_wifi_led "$board:green:router"
 	;;
+ex3700)
+	ucidef_set_led_default "power_a" "POWER (amber)" "$board:amber:power" "0"
```

Comment:
Drop this LED here, it is already handled by diag.sh.

I would suggests to use the green:power LED in diag.sh. Everything reddish indicates an error to me. You can drop the ```default-state = "on";``` from the green power LED in dts afterwards.

In case the amber LED is is switched on by the bootloader, add a ```default-state = "off";``` to the amber power led in the DTS (http://lxr.free-electrons.com/source/Documentation/devicetree/bindings/leds/leds-gpio.txt).